mobly/suite_runner.py

```diff
@@ -326,6 +328,50 @@ def compute_selected_tests(test_classes, selected_tests):
   # The user is selecting some tests to run. Parse the selectors.
   # Dict from test_name class name to list of tests to execute (or None for all
   # tests).
+  test_class_name_to_tests = _parse_raw_test_selector(selected_tests)
+
+  # Now transform class names to class objects.
```

Comment:
This doesn't match what is described in the function docstring, where `test_classes` is a list of strings. Looking deeper into the code, the `test_classes` param is indeed a list of class objects, so the docstring should be fixed.

Reply:
Good catch. DONE.